ledger-service/db-backend/src/test/scala/com/digitalasset/http/dbbackend/QueriesSpec.scala

```diff
@@ -6,6 +6,7 @@ package com.daml.http.dbbackend
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+import scala.collection.immutable.Seq
 
 class QueriesSpec extends AnyWordSpec with Matchers with ScalaCheckDrivenPropertyChecks {
   "uniqueSets" should {
```

Comment:
I'd really appreciate a couple of example-based tests cases for `uniqueSets` so I can see what it actually does. It puzzles me.

Reply:
Maybe a quick colloquial would be even faster to understand: groupBy, but all the first values, then all the second values, then all the 3rd values...